### DISCUSSION TITLE
Question before starting a development

### DISCUSSION THREAD

**Discussion by JAlcaideH** [everysight-maverick/sdk#3] [General]
Hello, I have some programming skills but it is my first time developing Android Apps, so I am not an expert. My idea is to create an app that will have the maverick SDK and another SDK that will provide information of the roads. So I would like to then send this information to the maverick glasses in order to display it. My question is: 
Can I send information to the glasses through that application (vía bluetooth I guess) that is running on my phone? 
Does the maverick SDK works like that, or should I develop a specific application that will be installed on the glasses? 
Also, where can I found more information about the methods in the SDK, only with the examples or somewhere else?

I hope someone can help me with these questions. Thanks.

**Comment by c00lzerosoftware** [comment]
Hi, I'm new also and can't fully answer the question but I can tell you apparantly the way it works is that we'll be developing a companion app that the glass will pair with to aggegate and display your information.  There is no installation on the glass itself, only an app on the phone that wirelesly streams the data to the glass, per say. I would start by compiling a couple of the samples to see what I mean...all that displays is a logo until you pair to your app and then it displays your data with that app running in the background. Perhaps Everysight will help chime in on more specific answer to your question but that's the basic functionality as I understand and have tested so far.

**Reply by JAlcaideH** [reply]
Thanks!!

**Comment by feranf** [comment]
Hi,

The Maverick SDK is designed to be integrated **into your application**. This means you don't have to develop anything else but your own application.
The SDK enables **your application** to connect to the glasses and provides you with an API for rendering information on the glasses.
Since your application probably already has all the required information within, it should be relatively easy to implement your required HUD graphics.

Hope that helped :-)

**Reply by JAlcaideH** [reply]
Thank you!
